src/app/content/hooks/storeTextSize.ts

```diff
@@ -38,7 +38,11 @@ const hookBody: ActionHookBody<typeof setTextSize> = (services) => async() => {
   const value = textSize(state);
 
   if (typeof window !== 'undefined' && value !== null) {
-    window.localStorage.setItem(textResizerStorageKey, value.toString());
+    try {
```

Comment:
pretty sure you have to do this up on line 22 as well

Reply:
I wasn't able to get an error from line 22 in Firefox or Safari, but I put the try/catch around it just in case. The setItem call was causing the page to blow up when localStorage was disabled, and now fonts change normally.